lambda_backend/postman_collections/Bouldering - Route.postman_collection.json

```diff
@@ -33,7 +33,12 @@
           "bearer": [
             {
               "key": "token",
-              "value": "eyJraWQiOiJVa3o4cUVIV1FVazZ4dTl5M1VtTGJHT2hldUZZOWRxZWR4bU80YmNEU0VzPSIsImFsZyI6IlJTMjU2In0.eyJvcmlnaW5fanRpIjoiNmRlNWZkMmMtOTYzNS00ZTk0LThlZGUtYzJhNTAzZjE1YjgwIiwic3ViIjoiZThjYzQwNTItM2YxZS00MzgwLWFiNjEtNjM3MzVlYmMwMDUyIiwiZXZlbnRfaWQiOiI0N2M1ZjdjNS0zMzIzLTQyOTgtYmU5NS05NWUxMjg5NWRjMzMiLCJ0b2tlbl91c2UiOiJhY2Nlc3MiLCJzY29wZSI6ImF3cy5jb2duaXRvLnNpZ25pbi51c2VyLmFkbWluIiwiYXV0aF90aW1lIjoxNjIzODU1ODIyLCJpc3MiOiJodHRwczpcL1wvY29nbml0by1pZHAuYXAtc291dGhlYXN0LTEuYW1hem9uYXdzLmNvbVwvYXAtc291dGhlYXN0LTFfVWxTOFNWQ3V1IiwiZXhwIjoxNjIzOTQyMjIyLCJpYXQiOjE2MjM4NTU4MjIsImp0aSI6ImQ5YWFmMGU0LTFlYTYtNGNkNy04NzhhLTBjNDhkYWYxYTdjNSIsImNsaWVudF9pZCI6IjNncHRrdmpmcG9xaXVuZWptMnNpbmNqaWVmIiwidXNlcm5hbWUiOiJ5YXJraGluZXBoeW8ifQ.jbP4LNY9jI-OAPeU-dZlNBuOBTv_lhIWX9d8KvUAG00LfkJKp3erIPHsaPKo8MVNPY2Ia0DnpIha-pPn9aLu1cUj4XP9dxYIgScifgwjANS2sAM04AdtNSyPSfef_YRX4lHOxnilr7FnqhdRvGFhL3di1nOE46sT5q18stmdh6k93YHhMfMDNa4b1LZx21t1zanQ36GQTZQTu4YHFgEiKlnCRNjirbyT8Hr1fEl7K6XLgBbG-nEnnoh33-26HNjYVMXdo-DQbYIruCpa_H84Opf-Qfdn1p-G1c_-6ZMK31iOD1lN46krHX6p_IHq0jx_OZis7o0pw__anhHcIkSj3w",
+              "value": "{{AccessToken}}",
+              "type": "string"
+            },
+            {
+              "key": "password",
+              "value": "{{{{AccessToken}}}}",
```

Comment:
what does this nesting of brackets do lol

Reply:
Oh it allows AccessToken to be an environment variable inside Postman, so you don't have to keep updating manually for all of them

Reply:
Why is this nesting 4 levels though

Reply:
removed